### DISCUSSION TITLE
How "hide on mobile" works ?

### DISCUSSION THREAD

**Discussion by martinduris** [bootstrap-email/bootstrap-email#87] [Help]
Hi,

i had used editor 'https://editor.bootstrapemail.com/' where i had just add this code to body:

```
<div class="card d-desktop">
    <div class="card-body">
      <h5>xxx</h5>
    </div>
  </div>
```

I suppose this should be visible only "on large displays" - but when i send email and open it in mobile, it is visible.
What am i doing wrong ?
(i had found this issue: https://github.com/bootstrap-email/bootstrap-email/issues/18 but could not find answer)

thanks !

**Comment by stuyam** [comment]
I would guess that that would work, however you may want to wrap the card in a parent div that has the `d-desktop` class. Also there are some mobile email clients that don't support media queries and that this might not work 100% on.
